x/ccv/provider/types/msg.go

```diff
@@ -746,112 +763,133 @@ func ValidateField(nameOfTheField string, field string, maxLength int) error {
 	return nil
 }
 
+// TruncateString truncates a string to maximum length characters
+func TruncateString(str string, length int) string {
```

Comment:
Where is this used?

Reply:
@MSalopek I assume this is to be used in the migration code when we create the `MsgCreateConsumer` and `MsgUpdateConsumer` messages from the proposals.

Reply:
Indeed. There are TODOs in the migration code on Gaia. It's important to not halt the upgrade if some fields are too big.